helm/kubernetes-namespace-prep/templates/authenticator-rolebinding.yml

```diff
@@ -0,0 +1,23 @@
+---
+{{- if .Values.authnRoleBinding.create -}}
+{{- $g := (lookup "v1" "ConfigMap" .Values.authnk8s.namespace .Values.authnk8s.goldenConfigmap).data -}}
+apiVersion: {{ include "conjur-prep.rbac-api" . }}
+kind: RoleBinding
+metadata:
+  name: {{ .Values.authnk8s.namespace }}-conjur-authenticator-role-binding
+  labels: 
+    app.kubernetes.io/name: {{ .Values.authnk8s.namespace }}-authn-k8s
+    app.kubernetes.io/component: rolebinding
+    app.kubernetes.io/instance: {{ .Values.authnk8s.namespace }}-authn-k8s 
+    app.kubernetes.io/part-of: app-namespace-config 
+    app.kubernetes.io/managed-by: helm
```

Comment:
Is this needed? I think Helm adds this.

Reply:
I'm actually not sure, it was included in the issue examples

Reply:
Yeah, I think Helm dynamically adds its own version of this with a different label, but same info. We can delete this one.